docs/product-business-resource-planner-guide/src/main/asciidoc/optimizer-planning-problem-con.adoc

```diff
@@ -0,0 +1,53 @@
+[id='optimizer-planning-problem-con']
+= What is a planning problem?
+
+
+A planning problem has an optimal goal, based on limited resources and under specific constraints. Optimal goals can be any number of things, such as:
+
+* Maximized profits - the optimal goal results in the highest possible profit.
+* Minimized ecological footprint - the optimal goal has the least amount of environmental impact.
+* Maximized satisfaction for employees or customers - the optimal goal prioritizes the needs of employees or customers.
+
+The ability to achieve these goals relies on the number of resources available, such as:
+
+* The number of people
+* Amount of time
+* Budget
+* Physical assets, for example, machinery, vehicles, computers, buildings, etc
+
+Specific constraints related to these resources must also be taken into account, such as the number of hours a person works, their ability to use certain machines, or compatibility between pieces of equipment. 
+ 
+The Business Optimizer helps Java programmers solve constraint satisfaction problems efficiently. Under the hood, it combines optimization heuristics and metaheuristics with efficient score calculation.
+
+
+[id='optimizer-planning-problem-search-space-con']
+== A Planning Problem Has a Huge Search Space
```

Comment:
This section should technically be a separate module that you include in the assembly. Each module should almost always have only one heading (meaning one genuine = Heading).

Also, the title should be sentence case, as you've correctly done everywhere else (A planning problem has a huge search space).

Reply:
Forgot to change that one, thanks,